src/types/contextAction.ts

```diff
@@ -49,4 +49,13 @@ export type ContextAction =
   | 'redeemed'
   | 'sold item'
   | 'joined dice game'
-  | 'dice game settled';
+  | 'dice game settled'
+  // Farcaster
+  | 'registered a Farcaster ID'
+  | 'changed recovery address to'
```

Comment:
```suggestion
  | 'changed recovery address'
```
We should keep prepositions out of the action names and move them to the description for a particular language like `en`